lib/ansible/plugins/vars/sops.py

```diff
@@ -0,0 +1,460 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Contributer: Brandon Schlueter <b@schlueter.blue> [:schlueter]
+# Contributor: Julien Vehent <jvehent@mozilla.com> [:ulfr]
+# Contributor: Daniel Thornton <daniel@relud.com>
+# Contributor: Alexis Metaireau <alexis@mozilla.com> [:alexis]
+# Contributor: Rémy Hubscher <natim@mozilla.com> [:natim]
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    vars: sops
+    version_added: "2.6"
+    short_description: Loads vars encrypted with Mozilla's sops.
+    description:
+      - Reads sops configuration from config file in playbook dir or above it.
+      - For any file in a group or host vars directory matching a creation
+        rule, it is loaded using sops (potentially using a sops configuration
+        file closer to it).
+      - Files are decrypted by default. This can be disabled by setting the
+        ANSIBLE_SOPS_IGNORE environment variable to yes, on, or true. If decryption
+        failure should fail the playbook run, ANSIBLE_SOPS_REQUIRE may be set.
+'''
+
+import hashlib
+import os
+import re
+import subprocess
+import sys
+from base64 import b64decode
+from socket import gethostname
+
+import boto3
```

Comment:
CI failure due to no `try/except ImportError` block around import from outside of the Python standard library.